generators/client/templates/react/src/main/webapp/app/config/notification-middleware.spec.ts.ejs

```diff
@@ -196,9 +239,33 @@ describe('Notification Middleware', () => {
     expect(toastMsg).toContain('foo.creation');
   });
 
-  it('should not trigger an error toast message and return error for 40 response code', () => {
+  it('should not trigger an error toast message and return error for 401 response code', () => {
     expect(store.dispatch(LOGIN_REJECTED_ERROR).error.response.status).toEqual(401);
     expect((toastify.toast as any).error.called).toEqual(false);
     expect((toastify.toast as any).success.called).toEqual(false);
   });
+
+  it('should trigger an error toast message and return error for 400 response code', () => {
+    expect(store.dispatch(TITLE_ERROR).error.response.status).toEqual(400);
+    const toastMsg = (toastify.toast as any).error.getCall(0).args[0];
+    expect(toastMsg).toContain('Incorrect password');
+  });
+
+  it('should trigger an error toast message and return error for string in data', () => {
```

Comment:
`should trigger an error toast message and return error for string in data` is the same L254, L260 and L266

Reply:
good catch